src/style.css

```diff
@@ -1,5 +1,5 @@
 .blendMenu {
-  min-width: auto !important;
+  min-width: 125px !important;
```

Comment:
Going from flexible to fixed pixels is sometimes sad.  Maybe spend 5 min looking at this; if no solution, no problem.

Reply:
I haven't figured this yet, but will keep it in my notes in case another idea comes up. Thanks for the heads up.